Modules/Apps/GitHub/FriedbergSIDR.mkape

```diff
@@ -0,0 +1,15 @@
+Description: SIDR - A tool designed to parse Windows search artifacts from Windows 10 (and prior) and Windows 11 systems. 
```

Comment:
Propose two small changes for the filename and the description.

I would rename the module file to `SIDR_WindowsIndexSearchParser.mkape` or something like that to state the purpose of the module.

In the description I would use the exact name of the articat `...Windows index search artifacts ...` so we would find it when searching for that specific term.